Color Picker/ColorPickerScreen.swift

```diff
@@ -127,22 +127,149 @@ private struct BarView: View {
 	}
 }
 
+struct ColorInputView: View {
+    @EnvironmentObject private var appState: AppState
+    @State private var textColor: Color = .primary
+    @Binding var inputColorText: String
+    @Binding var isTextFieldFocused: Bool
+    let colorPanel: NSColorPanel
+    let textFieldFontSize: Double
+    let inputColorType: ColorFormat
+
+    var body: some View {
+        HStack {
+            // TODO: When I use `TextField`, add the copy button using `.safeAreaInset()`.
+            NativeTextField(
+                text: $inputColorText,
+                placeholder: inputColorType.rawValue,
+                font: .monospacedSystemFont(ofSize: textFieldFontSize, weight: .regular),
+                isFocused: $isTextFieldFocused,
+                textColor: textColor
+            )
+                .controlSize(.large)
+                .onChange(of: inputColorText) {
```

Comment:
Move the body out into a private function on the view. It makes it easier to see just the view logic.